### PR TITLE
Iii

### PR DESCRIPTION
**Pull request by tabreturn**
# New Port for iii

## Game Information
- **Title**: iii
- **URL**: https://yukonw.itch.io/iii

### CFW Tests
Ensure your game has been tested on all major CFWs:
- [x] AmberELEC
- [x] ArkOS
- [x] ROCKNIX
- [x] muOS

### Resolution Tests
Test all major resolutions:
- [ ] 480x320
- [x] 640x480
- [x] 720x720 (RGB30)
- [x] Higher resolutions (e.g., 1280x720)

## File Structure
- Your port should have the following structure:
  - portname/
    - port.json
    - README.md
    - screenshot.jpg
    - cover.jpg
    - gameinfo.xml
    - Port Name.sh
    - portname/
      - <portfiles here>

## Additional Resources
For an in-depth guide on creating a pull request, refer to: [PortMaster Game Packaging Guide](https://portmaster.games/packaging.html#creating-a-pull-request)

### CODE REVIEW

File: ports/iii/iii.sh [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+cd $GAMEDIR
+
+# Check for file existence before trying to manipulate them:
+[ -f "./gamedata/data.win" ] && mv gamedata/data.win gamedata/game.droid
+[ -f "./gamedata/game.win" ] && mv gamedata/game.win gamedata/game.droid
+[ -f "./gamedata/game.unx" ] && mv gamedata/game.unx gamedata/game.droid
+
+# Delete unneeded files
+rm -f ./gamedata/*.exe
```
Comment (Cebion) [inline ports/iii/iii.sh:38]: Does this make more sense? Only run if the .exe files are present?
```suggestion
[ -f ./gamedata/*.exe ] && rm -f ./gamedata/*.exe
```

**Comment by Cebion** [comment]
Missing gameinfo.xml

File: ports/iii/gaminfo.xml
Comment (kloptops) [inline ports/iii/gaminfo.xml:1]: Should be named `gameinfo.xml`

**Review by kloptops — CHANGES_REQUESTED** [review]

File: ports/iii/gaminfo.xml
Comment (tabreturn) [reply ports/iii/gaminfo.xml:1]: woops, fixed

**Review by kloptops — APPROVED** [review]

**Review by Cebion — APPROVED** [review]

**Pull request merged**